README.md

```diff
@@ -0,0 +1,67 @@
+<h1 align="center">Welcome to bedrock-payment-paypal 👋</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000" />
+  <a href="https://github.com/digitalbazaar/bedrock-payment-paypal#readme">
+    <img alt="Documentation" src="https://img.shields.io/badge/documentation-yes-brightgreen.svg" target="_blank" />
+  </a>
+  <a href="https://github.com/digitalbazaar/bedrock-payment-paypal/graphs/commit-activity">
+    <img alt="Maintenance" src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" target="_blank" />
+  </a>
+</p>
+
+> This Plugin provides PayPal functionality to bedrock-payment.
+
+### 🏠 [Homepage](https://github.com/digitalbazaar/bedrock-payment-paypal#readme)
+
+## Install
+
+```sh
+npm install --save bedrock-payment-paypal
+```
+
+## Configure
+
+```js
+const {config} = require('bedrock');
+
+config.paypal = config.paypal || {};
+config.paypal.api = 'https://api.sandbox.paypal.com';
+config.paypal.clientId = 'your-client-id';
+config.paypal.secret = 'your-secret';
+// this will be the name that shows up in the papypal order.
+config.paypal.brandName = 'your-company-name';
+// this can be GET_FROM_FILE, NO_SHIPPING, & SET_PROVIDED_ADDRESS
+// it defaults to NO_SHIPPING.
+// You shouldn't have to set it at all.
+config.paypal.shippingPreference = 'NO_SHIPPING';
+```
+If you are concerned about your PayPal secret being exposed on github you can use
+an environment variable to store it:
+
+```sh
+export paypal_secret=your-secret-key
+export paypal_client_id=your_client_id
+```
+then use it in the config file
+
+```js
+config.paypal.clientId = process.env.paypal_client_id;
+config.paypal.secret = process.env.paypal_secret; 
+```
+
+## Author
+
+👤 **Digital Bazaar, Inc.**
+
+* GitHub: [@digitalbazaar](https://github.com/digitalbazaar)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!<br />Feel free to check [issues page](https://github.com/digitalbazaar/bedrock-payment-paypal/issues).
+
+## Show your support
+
+Give a ⭐️ if this project helped you!
+
+***
+_This README was generated with ❤️ by [readme-md-generator](https://github.com/kefranabg/readme-md-generator)_
```

Comment:
Some pushback on using this tool:
- This README is different than every other project we have.
- Why is there HTML output from a 'md' generator?
- The version badge won't auto-update.  We should avoid using things like that.
- I guess the emoji use is cute?  Seems like a bit much.

Perhaps we could just use the similar but simpler https://github.com/RichardLitt/standard-readme style like in https://github.com/digitalbazaar/crypto-ld?

Reply:
@davidlehn that guys a yeoman template for standard read me generation: https://github.com/RichardLitt/generator-standard-readme

Reply:
@davidlehn  I removed the html parts and I installed the standard read me generator. I'm not sure if I will get time to refactor this README, but I am a yeoman template away from it in the future. Removed the HTML here: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/3da073cedb3ac354b32ddc7343c6996e66046149

Reply:
I think copy and and modify with any editor can get close enough.  No reason to involve generators or other tools for a readme.

Reply:
@davidlehn used the generator and added a standard README: https://github.com/digitalbazaar/bedrock-payment-paypal/pull/2/commits/79df045d67ce18b90272d328b40c102113ca9b57